test/EVMHost.cpp

```diff
@@ -35,17 +35,17 @@ using namespace dev;
 using namespace dev::test;
 
 
-evmc::vm* EVMHost::getVM(string const& _path)
+evmc::VM* EVMHost::getVM(string const& _path)
 {
-	static unique_ptr<evmc::vm> theVM;
+	static unique_ptr<evmc::VM> theVM;
```

Comment:
The `evmc::VM` itself works as unique ptr, so it can be used directly here.

Reply:
Can you push a commit simplifying the entire function?

Reply:
Can you create a PR changing this after this PR is merged?